Classes/UT3Paladin.uc

```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright � 2008-2009 Wormbo
- * Copyright � 2008-2009, 2014 GreatEmerald
- * Copyright � 2012 100GPing100
+ * Copyright © 2008-2009 Wormbo
+ * Copyright © 2008-2009, 2014 GreatEmerald
+ * Copyright © 2012-2017 Luís 'zeluisping' Guimarães <zeluis.100@gmail.com>
```

Comment:
From what I recall, copyright law requires you to list years when work has been done on a file. Since there was no work done in between 2012 and 2017, these should all be `2012, 2017` instead of `2012-2017`. Also, in the cases where HellDragon found values, please add his copyright as well.